tests/unit/conditions/test_return_value.py

```diff
@@ -142,6 +143,23 @@ def test_return_value_test_with_context_variable_cant_run_eval():
         test.eval(0)
 
 
+def test_return_value_test_with_resolved_context():
+    test = ReturnValueTest(comparator="==", value=":foo")
+    context = {":foo": 1234}
+
+    resolved = test.with_resolved_context(**context)
+    assert resolved.comparator == test.comparator
+    assert resolved.index == test.index
+    assert resolved.value == _resolve_context_variable(test.value, **context)
```

Comment:
s far as the test goes, I assume you just want to ensure the context was used? Basically ensure that the resolution occurs correctly - since if we use the same function as the underlying resolution, the "wrong" values could be equal to each other.

```suggestion
    assert resolved.value == context[test.value]
```

Reply:
I originally had your suggestion, but I thought that that would be a less pure unit test.

Reply:
Definitely not a blocker, and `test_context` does a lot of the resolution checking.

Approving either way - up to you.